src/components/contentIpfs/component/link/index.tsx

```diff
@@ -1,12 +1,30 @@
 import GatewayContent from '../gateway';
+import TextMarkdown from 'src/components/TextMarkdown';
 
-function LinkHttp({ content, preview }) {
-  return (
-    <>
-      <div>{content}</div>
-      {preview && <GatewayContent url={content} />}
-    </>
-  );
+const EMBEDABLE_HOSTS = [
```

Comment:
could you clarify what does this sites can do?

Reply:
be rendered on website. others no